cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -64,6 +64,16 @@ spec:
       labels:
         k8s-app: kube-proxy
     spec:
+      initContainers:
+      - name: touch-lock
+        image: busybox
```

Comment:
@ixdy ^

I think we have a gcr.io image of it somewhere. The hard question is security updates for that one @timstclair